scripts/make_fake_pm_data.js

```diff
@@ -0,0 +1,134 @@
+    let sumGreenhouseGas = 0;
+
+    let dataEUI = [];
+    let dataEnergyStar = [];
+    let dataGreenhouseGas = [];
+
+    dataset.map(building => {
+        sumEUI += building.EUI;
+        sumEnergyStar += building.ENERGY_STAR_SCORE;
+        sumGreenhouseGas += building.GREENHOUSE_GAS_EMISSIONS;
+
+        dataEUI.push(building.EUI);
+        dataEnergyStar.push(building.ENERGY_STAR_SCORE);
+        dataGreenhouseGas.push(building.GREENHOUSE_GAS_EMISSIONS);
+    })
+
+    let meanEUI = Math.round((sumEUI / dataset.length))
+    let meanEnergyStar = Math.round((sumEnergyStar / dataset.length))
+    let meanGreenhouseGas = Math.round((sumGreenhouseGas / dataset.length))
+
+    let stdDevEUI = standardDeviation(meanEUI, dataEUI, dataset.length)
+    let stdDevEnergyStar = standardDeviation(meanEnergyStar, dataEnergyStar, dataset.length)
+    let stdDevGreenhouseGas = standardDeviation(meanGreenhouseGas, dataGreenhouseGas, dataset.length)
+
+    let normalizedBuildingsObj = dataset.map(building => {
+        return {
+            ...building,
+            latitude: shiftCoordinates(building.LAT_LONG[0], 3),
+            longitude: shiftCoordinates(building.LAT_LONG[1], -3),
+            city: "Springfield",
+            year: 2016,
+            reported_address: building.PROPERTY_ADDRESS,
+            property_type: building.PRIMARY_USE,
+            property_name: building.NOTES,
+            numunits: randomNumber(1, 100),
+            numfloors: Number(building.FLOORS),
+            numbuildings: Number(building.PROPERTY_BUILDINGS),
+            yearbuilt: Number(building.YEAR_BUILT),
+            reported_gross_floor_area: Number(building.APPROX_BUILDING_AREA.split(",").join("")),
+            site_eui: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI)),
+            source_eui: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI)),
+            energy_star_score: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI)),
+            total_ghg_emissions: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI)),
+            total_ghg_emissions_intensity: (Math.sign(randomNumberOfStandardDeviations(-1, 1)) === -1) ? (Math.round(meanEUI + (randomNumberOfStandardDeviations(-1, 2) * stdDevEUI))) : (Math.round((randomNumberOfStandardDeviations(-1, 1) * stdDevEUI) + meanEUI))
```

Comment:
So these don't really depend on the current value of the generated object at all. We could probably not generate the first ones (the `randomNumber(0,100)`) and just ... there's something that can be clearer here. Lost track of how this is working. Not entirely sure on this one yet.